src/NuGet.Clients/PackageManagement.UI/Resources/UpgradeReport.xslt

```diff
@@ -0,0 +1,322 @@
+          <tr>
+            <td class="issueCell">
+              <span>
+                <xsl:value-of select="@Name" />
+              </span>
+            </td>
+          </tr>
+        </xsl:for-each>
+        <xsl:choose>
+          <xsl:when test="$excludedPackageCount = 0">
+            <tr>
+              <td class="issueCell">
+                No packages were excluded.
+              </td>
+            </tr>
+          </xsl:when>
+        </xsl:choose>
+      </table>
+    </xsl:for-each>
+  </xsl:template>
+
+  <!-- Project Details Template -->
+  <xsl:template match="Projects" mode="ProjectDetails">
+    <xsl:for-each select="Project">
+      <xsl:variable name="issuesCount" select="count(Issues/Issue)" />
+
+      <table>
+        <tr>
+          <th />
+          <th class="issueCell" _locID="DescriptionTableHeader">Description</th>
+        </tr>
+
+        <xsl:for-each select="Issues/Issue">
+          <xsl:sort select="@Level" order="descending" />
+          <xsl:apply-templates select="." />
+        </xsl:for-each>
+
+        <xsl:choose>
+          <xsl:when test="$issuesCount = 0">
+            <tr>
+              <td class="IconInfoEncoded" />
+              <td class="issueCell">
+                No issues were found.
+              </td>
+            </tr>
+          </xsl:when>
+        </xsl:choose>
+
+
+      </table>
+    </xsl:for-each>
+  </xsl:template>
+
+  <!-- Document, matches "NuGetUpgradeReport" -->
+  <xsl:template match="NuGetUpgradeReport">
+    <!-- Output doc type the 'Mark of the web' which disabled prompting to run JavaScript from local HTML Files in IE -->
+    <!-- NOTE: The whitespace around the 'Mark of the web' is important it must be exact -->
+    <xsl:text disable-output-escaping="yes"><![CDATA[<!DOCTYPE html>
+<!-- saved from url=(0014)about:internet -->
+]]>
+    </xsl:text>
+    <html>
+      <head>
+        <meta content="en-us" http-equiv="Content-Language" />
+        <meta content="text/html; charset=utf-16" http-equiv="Content-Type" />
+        <title _locID="NuGetUpgradeReportTitle">
+          NuGet Upgrade Report
+        </title>
+        <style>
+          <xsl:text disable-output-escaping="yes">
+            <![CDATA[
+                    /* Body style, for the entire document */
+                    body
+                    {
+                        background: #F3F3F4;
+                        color: #1E1E1F;
+                        font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+                        font-size: 12pt;
+                        padding: 0;
+                        margin: 0;
+                    }
+
+                    /* Header1 style, used for the main title */
+                    h1
+                    {
+                        padding: 10px 0px 10px 10px;
+                        font-size: 21pt;
+                        background-color: #E2E2E2;
+                        border-bottom: 1px #C1C1C2 solid; 
```

Comment:
I expect that you've just duplicated these colors from existing html reports we generate?